packages/nodes-base/nodes/Microsoft/Storage/test/blob/create.test.ts

```diff
@@ -0,0 +1,72 @@
+import nock from 'nock';
+
+import { equalityTest, setup, workflowToTests } from '@test/nodes/Helpers';
+
+describe('Azure Storage Node', () => {
+	const workflows = ['nodes/Microsoft/Storage/test/blob/create.workflow.json'];
+	const workflowTests = workflowToTests(workflows);
+
+	beforeEach(() => {
+		// https://github.com/nock/nock/issues/2057#issuecomment-663665683
+		if (!nock.isActive()) {
+			nock.activate();
+		}
+	});
+
+	describe('should create blob', () => {
```

Comment:
Nitpick (and this applies for all the nested `describe` in the `.test.ts` files in this commit):
```suggestion
	it('should create blob', () => {
```